GoogleSignIn/Sources/GIDAuthentication.m

```diff
@@ -255,15 +234,7 @@ - (void)doWithFreshTokens:(GIDAuthenticationCompletion)completion {
                                       callback:^(OIDTokenResponse *_Nullable tokenResponse,
                                                  NSError *_Nullable error) {
     if (tokenResponse) {
-      [self willChangeValueForKey:NSStringFromSelector(@selector(accessToken))];
```

Comment:
We should maintain the same level of testing coverage here.  Is there some reason this is particularly difficult?

Reply:
This is not difficult. But these properties will change when OIDAuthentication `doWIthFreshTokens` method is invoked. That means we need to add new OCMocks. In the next project we will extract a new protocol and use fake s in unit tests. Then we will remove these mocks just added. I would prefer to avoid these back and forth changes.